python/lessons_help_wanted.py

```diff
@@ -0,0 +1,182 @@
+    all_help_wanted_issues = [] 
+
+    # Go through each repo
+    # Search for issues labeled "help wanted" or "good first issue"
+    
+    for r in repos_with_topic:
+
+        # Set up API request
+        org_name = r['org_name']
+        repo_name = r['repo_name']
+        url = "https://api.github.com/search/issues"
+        # lbl = "\"help wanted\",\"good first issue\""
+        query = f"repo:{org_name}/{repo_name} label:{issue_labels} state:open is:issue"
+        # query = f"repo:{org_name}/{repo_name} label:'help wanted','good first issue' state:open is:issue"
+
+        params = {
+                "q": query
+                }
+        
+        help_wanted_issues_req = requests.get(url, headers=headers, params=params)
+
+        if help_wanted_issues_req.status_code != 200:
+            print(f"Failed to retreive issues: {help_wanted_issues_req.status_code}, {help_wanted_issues_req.json()}")
+            return help_wanted_issues_req.status_code
+        help_wanted_issues = help_wanted_issues_req.json()['items']
+
+        # Process data 
+      
+   
+        for i in help_wanted_issues:
+
+            # Create copy of the repo dict
+            hw_issue = r.copy() 
+
+            # Create csv string of label names 
+            labels = [x['name'] for x in i['labels']]
+            labels = ", ".join(labels)
+
+            # Convert dates to human readable format
+            created_at = datetime.strptime(i['created_at'], "%Y-%m-%dT%H:%M:%SZ").strftime("%d %B %Y")
+            updated_at = datetime.strptime(i['updated_at'], "%Y-%m-%dT%H:%M:%SZ").strftime("%d %B %Y")
+
+            # Add fields for this issue to the dict
+            hw_issue['issue_url'] = i['html_url']
+            hw_issue['issue_title'] = i['title']
+            hw_issue['created_at'] = created_at
+            hw_issue['updated_at'] = updated_at
+            hw_issue['labels'] = labels
+
+            # Add this dict to list of all help wanted issues 
+            all_help_wanted_issues.append(hw_issue)
+
+    count_issues = len(all_help_wanted_issues)
+    print(f"Found {count_issues} issues.\n")
+    return all_help_wanted_issues 
+
+# Initialize empty list to hold all issues 
+all_issues = []
+
+# Loop through all orgs, get repos and issues
+# Add issues to all_issues list
+
+# Note rate limit is 30 requests per minute for search
+# https://docs.github.com/en/rest/search/search?apiVersion=2022-11-28
+# Script will need modification if any LPs exceed 30 repos to query.
+# DC is currently at 22.
+
+for i in ALL_ORGS:
+    try:
+        repos = get_repos_by_topic(i, topics)
+        issues = get_help_wanted_issues(repos, labels)
+        all_issues += issues
+    except Exception as e:
+        print(f"Error with repo: '{i}', '{e}'")
+        continue
+    print("Pausing 60 seconds to reset GH API rate limit...\n")
+    time.sleep(60)
+
+print(len(all_issues))
+
+
+filename = 'lessons_help_wanted.json'
+
+with open(filename, 'w') as file:
```

Comment:
This should be wrapped in a try except to confirm the file can be opened and written to, probably with FileNotFoundError and/or IOError